examples/tti/tti_operators.py

```diff
@@ -21,58 +21,64 @@ class ForwardOperator(Operator):
     `Bhaskara` uses a rational approximation.
     """
     def __init__(self, model, src, damp, data, time_order=2, spc_order=4, save=False,
-                 trigonometry='normal', **kwargs):
+                 trigonometry='normal', u_ini=None, **kwargs):
         nt, nrec = data.shape
         nt, nsrc = src.shape
         dt = model.get_critical_dt()
         u = TimeData(name="u", shape=model.get_shape_comp(),
                      time_dim=nt, time_order=time_order,
-                     space_order=spc_order,
+                     space_order=spc_order/2,
```

Comment:
This is so that the discretization is consistent with acoustic.  We apply d/dx to d/dx so using `spc_order/2` makes the second derivative (Laplacian) of order `spc_order`.